### PR TITLE
Added SDF.nim

### PR DESCRIPTION
**Pull request by Halsys**

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -5264,5 +5264,22 @@
     "description": "ISAAC PRNG implementation on Nim",
     "license": "MIT",
     "web": "https://github.com/pragmagic/isaac/"
+  },
+  {
+    "name": "SDF.nim",
```
Comment (dom96) [inline packages.json]: Remove the ``.nim``, the name needs to match your package name.

Comment (Halsys) [inline packages.json]: I made the change but just a small question. Would `SDF.nim.nimble` be an also acceptable package filename? Sorry.

Comment (Halsys) [inline packages.json]: Or does this have a similar effect like dashes in names?

Comment (dom96) [inline packages.json]: I think it does. You would then need to rename your module to ``SDF.nim.nim`` and import it via ``import SDF.nim``, I dunno, maybe that works. Best way to check is to test it.

I'll merge this for now.

**Pull request merged**